src/contentmanagerside.cpp

```diff
@@ -80,18 +80,12 @@ void ContentManagerSide::setContentManager(ContentManager *contentManager)
     mp_contentManager = contentManager;
     connect(mp_languages, &KiwixChoiceBox::choiceUpdated,
             this, [=](QStringList values) {
-                if (values[0] == "all") {
-                    values = QStringList();
-                }
```

Comment:
All the things about the `all` or `*` is important. It would be better to have a specific commit instead of hidding it behind a "Also made a few structural changes making everything consistent."

Reply:
Done